tide_media.install

```diff
@@ -378,3 +378,23 @@ function tide_media_update_8011() {
     $type->delete();
   }
 }
+
+/**
+ * Clean up video options from entity browsers.
+ */
+function tide_media_update_8012() {
+  $entity_browser_ids = [
+    'c3bc9243-a506-4d08-8500-1cad2f2fc66b' => 'tide_media_browser_iframe',
```

Comment:
Do these change from database to database?

Reply:
@anthony-malkoun .yeah.  all client sites are using those uuid, they originally came from tide_media, thus they are same. i think it could be used in automation for fixing the issue.

Reply:
I had created `1.7.0-p2`  tag.